src/swap/saga.test.ts

```diff
@@ -518,8 +453,8 @@ describe(swapSubmitSaga, () => {
         swapExecuteTxId: 'id-swap/saga-Swap/Execute',
         quoteToUserConfirmsSwapElapsedTimeInMs: 2_500,
         quoteToTransactionElapsedTimeInMs: 10_000,
-        estimatedBuyTokenUsdValue: 0.005,
-        estimatedSellTokenUsdValue: 0.01,
+        estimatedBuyTokenUsdValue: 100,
```

Comment:
just checking that it's expected that these values changed? i

Reply:
Yes, previously it was using the values attached to the received `SwapTransaction`, but now it uses the swap input (which depends on the received quote price).